Plugins/Flow.Launcher.Plugin.Sys/Main.cs

```diff
@@ -192,6 +192,156 @@ private List<Result> Commands()
                     }
                 },
                 new Result
+                {
+                    Title = "Explorer",
+                    SubTitle = context.API.GetTranslation("flowlauncher_plugin_sys_explorer"),
+                    IcoPath = "Images\\explorer.png",
+                    Glyph = new GlyphInfo (FontFamily:"/Resources/#Segoe Fluent Icons", Glyph:"\xec50"),
+                    Action = c =>
+                    {
+                        {
+                            System.Diagnostics.Process.Start("explorer.exe", "");
+                        }
+
+                        return true;
+                    }
+                },
+                new Result
+                {
+                    Title = "My PC",
+                    SubTitle = context.API.GetTranslation("flowlauncher_plugin_sys_openmypc"),
+                    IcoPath = "Images\\mycomputer.png",
+                    Glyph = new GlyphInfo (FontFamily:"/Resources/#Segoe Fluent Icons", Glyph:"\xe7f4"),
+                    Action = c =>
+                    {
+                        {
+                            context.API.OpenDirectory("shell:mycomputerfolder");
```

Comment:
this kind of link doesn't really got supported by 3rd explorer? Is the one you are using support it?

Reply:
yeah, It has trouble with 3rd party file manager. I've tried a few things to work with third-party managers, but there's been a bit of a complicated problem. so I just simply remain only 'Open Recycle Bin' and 'Index Option'.